go.mod

```diff
@@ -6,6 +6,9 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/google/go-cmp v0.4.0
 	github.com/imdario/mergo v0.3.9
+	github.com/kr/text v0.2.0 // indirect
```

Comment:
It's unusual that indirect dependencies got added without a new direct dependency.  Can you try running `go mod tidy`?

Reply:
Ran it, didn't remove it.